projects/uitdatabank/docs/terms.md

```diff
@@ -1,7 +1,95 @@
 # Terms
 
-This page is still under construction. For now the documentation about terms can be found on the old documentation portal:
+The categorisation model of UiTdatabank is a standard aimed at simplifying data exchange and avoiding complex mapping procedures.
 
-* [Event types](https://documentatie.uitdatabank.be/content/uitdatabank/latest/categorisatie/type_aanbod.html)
-* [Event themes](https://documentatie.uitdatabank.be/content/uitdatabank/latest/categorisatie/combinatie_type_thema.html)
-* [Facilities](https://documentatie.uitdatabank.be/content/uitdatabank/latest/categorisatie/voorzieningen.html)
+The main purpose of the categorisation model is to categorise cultural and leisure activities. The categorisation model consists of 3 dimensions (`domains`):
```

Comment:
```suggestion
The main purpose of the categorisation model is to categorise cultural and leisure activities. The categorisation model consists of 3 domains:
```

It's just a suggestion, but I'm personally not sure what "dimensions" adds that "domains" does not communicate? If it's a term that's often used and adds some value I'm okay with leaving it in but I haven't heard it before in this context (of taxonomy terms).